### DISCUSSION TITLE
Value set in changelog parameter in testflight/pilot actions does not appear to TestFlight testers

### DISCUSSION THREAD

**Discussion by hilsenrat** [fastlane/fastlane#17525] [Q&A]
Hi!

I'm deploying an app to TestFlight using `pilot` action.
I'm want to add information for testers in the "What to Test" window, and I'm using `changelog` parameter for that cause. I'm doing it that way:
```
desc "Upload to Testflight"
  lane :build_and_testflight do |options|
    build
    if options[:testflight_changelog]
      puts "CHANGELOG IS: #{options[:testflight_changelog]}"
      testflight(
        changelog: "#{options[:testflight_changelog]}"
      )
    else
      puts "You must supply changelog by using testflight_changelog parameter!"
    end
  end
```

It's being invoked with this line: `bundle exec fastlane build_and_testflight --env AcrKidneyCheckReleaseStaging testflight_changelog:"This is a fake Release Note"`

The version uploaded successfully to TestFlight, but unfortunately the "What to Test" text is empty. 
The string I set for the test is `"Fake Release Notes"` (English, no special chars, etc.), and I don't get any error during the execution.

Am I missing something? Or should I open an issue regarding it?

Thanks! 🙏

**Comment by rogerluan** [comment]
Hey @hilsenrat 👋 

This is a known issue: https://github.com/fastlane/fastlane/issues/16129
Could you check if the workaround proposed works for you? https://github.com/fastlane/fastlane/issues/16129#issuecomment-672972631
Although this issue should be fixed, while we don't have a fix for it, having a working workaround would help a lot of users, but I'm not sure whether it works for everyone. 

Let me know! 🤗 Good luck!

**Reply by hilsenrat** [reply]
Hi @rogerluan, thanks for replying but unfortunately this WA didn't work for me (and for other fellows as well, at least from reading the few comments on that issue).
Is there any other things I can do to help investigate this issue?

Thanks again for the feedback.

**Reply by rogerluan** [reply]
I don't have any other suggestions unfortunately 😢 let me know if you managed to resolve this issue somehow. 🙏

**Comment by axelson** [comment]
@hilsenrat Maybe you could try a workaround that was posted later: https://github.com/fastlane/fastlane/issues/16129#issuecomment-778457548
